proj.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np  # linear algebra
```

Comment:
Remove these comments, these are not helping much. Only use comments when it's a matter of like & death. Your code should be readable and self-explanatory.

Reply:
Comments removed & noted!